src/types.ts

```diff
@@ -49,6 +50,12 @@ type Hooks = {
 export interface State extends Options<any> {
   eventHandler?: any;
   hooks: Hooks;
+  log: {
+    debug: (message: string) => unknown;
+    info: (message: string) => unknown;
+    warn: (message: string) => unknown;
+    error: (message: string) => unknown;
+  };
```

Comment:
Copied from https://github.com/octokit/core.js/blob/124bf5a64a1aef1fe0591ccff9b8cca87663cdbf/src/types.ts#L15-L20

Reply:
I think it'd be good to have these as an exported interface marked as optional, for reuse (I could see myself doing `const mockLogger: Logger = { ... }` in tests related to this package), and then do `Required<>` here.

Reply:
Done, I added a `Logger` interface.